apps/base/admin.py

```diff
@@ -1,5 +1,11 @@
 from django.contrib import admin
 
+from import_export.admin import ImportExportMixin
+
 
 class TimeStampedAdmin(admin.ModelAdmin):
     exclude = ('created_at', 'modified_at',)
+
+
+class ImportExportTimeStampedAdmin(ImportExportMixin, admin.ModelAdmin):
+    exclude = ('created_at', 'modified_at',)
```

Comment:
We are already using `TimeStampedAdmin` for all of our `ModelAdmin` classes. 

This `excludes` means that fields `created_at` and `updated_at` will not be displayed in edit view of any row. It in any ways has no relation with csv import export. For that, we define `exclude` and `fields` in `ModelResource` class.

I hope you got my point.